Gemfile

```diff
@@ -97,7 +97,7 @@ gem "rufus-scheduler",                "~>3.1.3",   :require => false
 gem "savon",                          "~>2.2.0",   :require => false  # Automate uses this for simple SOAP Integration
 gem "snmp",                           "~>1.2.0",   :require => false
 gem "uglifier",                       "~>2.7.1",   :require => false
-
+gem "where-or"
```

Comment:
Can you put this right next to the `rails` entry? That should force a conflict, so I can't miss it.